openassessment/xblock/test/test_staff_info.py

```diff
@@ -102,7 +102,7 @@ def test_course_student_debug_info(self, xblock):
         # If we ARE course staff, then we should see the debug info
         xblock.xmodule_runtime.user_is_staff = True
         resp = self.request(xblock, 'render_student_info', json.dumps({}))
-        self.assertIn("couldn\'t find a response for this student.", resp.decode('utf-8').lower())
+        self.assertIn("couldn\'t find a response for this learner.", resp.decode('utf-8').lower())
```

Comment:
Are these user-visible strings? Not sure if we need to fix punctuation, capitalization.